app/routes/_index.tsx

```diff
@@ -1,22 +1,90 @@
-import { Card, CardGroup, CardHeader } from "@trussworks/react-uswds";
+import {
+    Breadcrumb,
+    BreadcrumbBar,
+    BreadcrumbLink,
+    Card,
+    CardGroup,
+    CardHeader,
+    Header,
+    Label,
+    Link,
+    Select,
+} from "@trussworks/react-uswds";
+import React from "react";
 import { mergeMeta } from "~/lib/merge-meta";
 
 export const meta = mergeMeta(({ parentTitle }) => [{ title: `Meeting Spaces • ${parentTitle}` }]);
 
 export default function Index() {
     return (
-        <CardGroup>
-            <Card>
-                <CardHeader>
-                    <h1 className="usa-card__heading">Meeting Spaces</h1>
-                    <p className="font-sans text-base-darker">
-                        Austin Public Library Meeting Spaces are
-                        <strong className="font-bold"> free of charge</strong> and ideal for
-                        discussion groups, panels, and lectures. Both paper and online reservation
-                        requests are timestamped and processed in the order they are received.
-                    </p>
-                </CardHeader>
-            </Card>
-        </CardGroup>
+        <>
+            <div className="flex justify-center">
+                <div className="min-w-[30rem] max-w-[62rem]">
+                    <CardGroup>
+                        <Card>
+                            <Header>
+                                <div className="min-w-[30rem] max-w-[62rem] object-contain">
+                                    <img
+                                        alt="Public Meeting Room"
+                                        className="rounded-t-md"
+                                        loading="lazy"
+                                        src="/meetingSpacesHeader.jpg"
+                                    />
+                                </div>
+                                <BreadcrumbBar className="ml-2">
+                                    <Breadcrumb>
+                                        <BreadcrumbLink href="#">
+                                            <span>Home</span>
```

Comment:
You can get font / font size directly from figma by double clicking on text and use that to match styling

Reply:
That'll be something we can talk about; the implementation for the Breadcrumb component here is how USWDS defines using it, which is the guideline we're going by.

Reply:
@sterlingmuller tell me more about what you were thinking here

Reply:
Sorry, I didn't mean for that comment to be related to the breadcrumb components, just meant in general for fonts / widths and such. Wasn't sure if Nate new you can pull the styling from Figma